iris_grib/tests/integration/save_rules/test_save_hybrid_height.py

```diff
@@ -0,0 +1,91 @@
+# (C) British Crown Copyright 2018, Met Office
+#
+# This file is part of iris-grib.
+#
+# iris-grib is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# iris-grib is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with iris-grib.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Unit tests for :func:`iris_grib._save_rules.data_section`.
+
+"""
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+# import iris_grib.tests first so that some things can be initialised before
+# importing anything else
+import iris_grib.tests as tests
+
+
+import iris
+from iris.cube import Cube
+from iris import sample_data_path
```

Comment:
Apparently this isn't the right way to import the sample data.  We will have to figure this out.

Reply:
This is not importing the sample data, only the iris 'sample_data_path' method.

The sample data would need to be installed by Travis.yml
It's pretty heavyweight + I don't think we want to start doing that for iris-grib, so I'll look for some other test data ...